src/Symfony/Bridge/Twig/composer.json

```diff
@@ -22,7 +22,7 @@
     "require-dev": {
         "symfony/asset": "~2.7",
         "symfony/finder": "~2.3",
-        "symfony/form": "~2.7.10|~2.8.3",
```

Comment:
This would allow, for example, `2.7.0` and `2.8.0` which would be used when running the CI job with `deps=low` which would then fail.